src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -1145,6 +1146,13 @@ void VM_Version::get_processor_features() {
       }
       FLAG_SET_DEFAULT(UseChaCha20Intrinsics, false);
   }
+#else
+  // No support currently for ChaCha20 intrinsics on 32-bit platforms
+  if (UseChaCha20Intrinsics) {
+      warning("Support for ChaCha20 intrinsics not available on this CPU.");
```

Comment:
Maybe we can adapt the message to follow the convention of the other warning messages for intrinsics like AES
https://github.com/openjdk/jdk/blob/2611a49ea13ee7a07f22692c3a4b32856ec5898f/src/hotspot/cpu/x86/vm_version_x86.cpp#L1062-L1065
or CRC32C
https://github.com/openjdk/jdk/blob/2611a49ea13ee7a07f22692c3a4b32856ec5898f/src/hotspot/cpu/x86/vm_version_x86.cpp#L1116-L1118

```suggestion
      warning("ChaCha20 intrinsics are not available on this CPU.");
```

Though I see that not all existing warning messages are consistent.

Anyway, the bailout looks good to me.

Reply:
+1 on changing the message to "ChaCha20 intrinsics are not available on this CPU."

Reply:
Makes sense to me.  Will fix.